Sources/Brave/Frontend/Settings/SettingsViewController.swift

```diff
@@ -656,11 +656,24 @@ class SettingsViewController: TableViewController {
     return Section(
       header: .title(Strings.security),
       rows: [
-        .boolRow(
-          title: Strings.Privacy.browserLock,
+        Row(
+          text: Strings.Privacy.browserLock,
           detailText: Strings.Privacy.browserLockDescription,
-          option: Preferences.Privacy.lockWithPasscode,
-          image: UIImage(braveSystemNamed: "leo.biometric.login")),
+          image: UIImage(braveSystemNamed: "leo.biometric.login"),
+          accessory: .view(SwitchAccessoryView(initialValue: Preferences.Privacy.lockWithPasscode.value, valueChange: { isOn in
```

Comment:
`boolRow` has an argument that takes a `toggleValue` and `valueChange` argument to do just this fyi

Reply:
But no image argument, so that's why I used this one.